src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -1,15 +1,31 @@
 import { router, useLocalSearchParams } from 'expo-router';
-import React from 'react';
-import { View, Text } from 'react-native';
-import { TouchableOpacity } from 'react-native-gesture-handler';
+import React, { useEffect, useState } from 'react';
+import { Image, View, Text, TouchableOpacity, ScrollView } from 'react-native';
 
 import styles from './styles';
+import LineBig from '../../../../../assets/line-big.svg';
+import LineSmall from '../../../../../assets/line-small.svg';
 import RightArrow from '../../../../../assets/right-arrow.svg';
-import { updateCaseStatus } from '../../../../supabase/queries/cases';
-import { CaseUid, Eligibility } from '../../../../types/types';
+import Checkbox from '../../../../Components/Checkbox/checkbox';
+import {
+  updateCaseStatus,
+  getCaseById,
+} from '../../../../supabase/queries/cases';
+import { Case, CaseUid, Eligibility } from '../../../../types/types';
 
 export default function EligibilityForm() {
   const { caseUid } = useLocalSearchParams<{ caseUid: CaseUid }>();
+  const [caseData, setCaseData] = useState<Case>();
+
+  useEffect(() => {
+    async function fetchCaseData() {
```

Comment:
Move the `fetchCaseData` function creation outside of the useEffect. Otherwise it'll reinitialize this function during each useEffect trigger.